webrtc.html

```diff
@@ -1862,6 +1868,39 @@
         </dl>
       </section>
     </section>
+    <section>
+      <h3>Session Negotiation Model</h3>
+      <p>Many changes to the PeerConnection state will require communication with
+        the remote side via the signaling channel. The app can be kept informed
```

Comment:
Suggest adding "in order to have the desired effect" at the end of the sentence. IE you can add tracks as much as you want, but the likely desired effect is to have them sent to the other side, and that won't happen until negotiation is done.